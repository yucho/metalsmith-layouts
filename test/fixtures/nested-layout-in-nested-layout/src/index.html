---
layout: third.hbs
---
<p>The home page</p>
