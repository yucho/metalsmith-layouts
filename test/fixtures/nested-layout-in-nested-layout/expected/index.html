<!DOCTYPE html>
<html>
  <head>
    <title>Home</title>
  </head>
  <body>
    <h1>Heading</h1>
<h2>Subeading</h2>
<p>The home page</p>



  </body>
</html>
